get_libwallet_api.sh

```diff
@@ -146,7 +160,7 @@ fi
 # since filename conflict (random.c.obj)
 # for Linux, we use libunbound shipped with the system, so we don't need to build it
 
-if [ "$platform" != "linux32" ] && [ "$platform" != "linux64" ]; then
+if [ "$platform" != "linux32" ] && [ "$ANDROID" == true ] || [ "$platform" != "linux64" ]; then
```

Comment:
Is this correct? Looks like linux32 would slip though here when $ANDROID is false.